src/amo/components/Collection/index.js

```diff
@@ -426,6 +426,7 @@ export class CollectionBase extends React.Component<InternalProps> {
       <ConfirmButton
         buttonType="cancel"
         className="Collection-delete-button"
+        id="Collection-delete-button"
```

Comment:
I suppose it would make sense to use a shared variable here like `id={className}`